packages/signals/browser-signals/src/core/client/__tests__/redact.test.ts

```diff
@@ -0,0 +1,36 @@
+import { redactJsonValues } from '../redact'
+
+describe('redactJsonValues', () => {
+  it('should redact string values in an object', () => {
+    const obj = { name: 'John Doe', age: '30' }
+    const expected = { name: 'XXXX XXX', age: '99' }
+    expect(redactJsonValues(obj)).toEqual(expected)
+  })
+
+  it('should redact string values in a nested object', () => {
+    const obj = { user: { name: 'Jane Doe', age: '25' }, active: true }
+    const expected = {
+      user: { name: 'XXXX XXX', age: '99' },
```

Comment:
Per our prev. discussion, I think we should do that log 10 thing you mentioned ... I would avoid preserving string length or spaces like mobile -- worried that it deanonymizes -- just "XXX" for any string and "99" for any number is my preference.

for example:

city: XXX XXXX (New York), etc